src/components/implementation/tests/micro_mbox_client/mbox_client.c

```diff
@@ -61,13 +61,13 @@ void cos_init(void *arg)
 	rdtscll(start);
 	for (i=1; i<=j; i++) {
 		if (i == j)    rdtscll(end);
-		d = cbufp_alloc(sz, &cb1);
+		d = cbuf_alloc(sz, &cb1, 0);
 		if (!d) goto done;
-		cbufp_send(cb1);
+		cbuf_send(cb1);
 		rdtscll(end);
 		((u64_t *)d)[0] = end;
 		ret = twritep(cos_spd_id(), serv, cb1, sz);
-		cbufp_deref(cb1); 
+		cbuf_free(cb1); 
```

Comment:
You got rid of `cbuf_deref` here (replaced with `cbuf_free`), but kept `cbuf_send_deref`.  I suggest you change `cbuf_send_deref` into `cbuf_send_free`.